rootston/output.c

```diff
@@ -384,6 +400,20 @@ static bool has_standalone_surface(struct roots_view *view) {
 	return true;
 }
 
+static void surface_send_frame_done(struct wlr_surface *surface, double lx,
+		double ly, float rotation, void *_data) {
```

Comment:
Why isn't `void *_data` typed?

Reply:
Because it's a callback for the `*_for_each_surface` functions.